tests_disabled.json

```diff
@@ -209,10 +209,6 @@
             "name": "t_frang.test_concurrent_connections.ConcurrentConnections.test_clear_client_connection_stats",
             "reason": "Disabled by issue #1740"
         },
-        {
-            "name": "t_frang.test_client_body_and_header_timeout",
-            "reason": "Disabled by issue #1762"
-        },
```

Comment:
Please add these tests to `tests_disabled_tcpseg.json` with reason - "These tests should not be run with TCP segmentation.".

Reply:
Fixed